packages/components/select/src/select.vue

```diff
@@ -528,7 +528,13 @@ export default defineComponent({
       values: placements,
       default: 'bottom-start',
     },
-    ariaLabel: String,
+    /**
+     * @description native input aria-label
+     */
+    label: {
```

Comment:
```suggestion
    ariaLabel: {
```

Perhaps you misunderstood my meaning. It is more reasonable to use `ariaLabel` for the ElSelect component.

The `label` is an attribute of ElInput

Reply:
I just thought that `label` is used on ElInput so ElSelect should use it as well, for consistency. But I can change it back